config/redirects.txt

```diff
@@ -271,7 +271,7 @@
 (.*)\.md(.*) -> $1$2
 ^\/introduction\/introduction\/?(\?.+)?(#.+)?$ -> /learn/welcome/introduction/$1$2
 ^\/?(\?.+)?(#.+)?$ -> /learn/welcome/introduction/$1$2
-^\/installing\/gettingStarted\/?(\?.+)?(#.+)?$-> /learn/getting-started/raspberrypi3/nodejs/$1$2
+^\/installing\/gettingStarted\/?(\?.+)?(#.+)?$-> /learn/getting-started/raspberrypi5s/nodejs/$1$2
```

Comment:
On lines 57 and 58 you have `/raspberrypi5/` but on line 274 you have `/raspberrypi5s/` ?

Reply:
Thanks @lurch I saw that after merge too since the redirects were working. Fixed now https://github.com/balena-io/docs/pull/2818

Ps: Great work on the RPi5 by RaspberryPi Foundation - glad to have it supported day 0